ngraph/python/CMakeLists.txt

```diff
@@ -73,11 +77,35 @@ file(GLOB_RECURSE SOURCES src/pyngraph/*.cpp)
 pybind11_add_module(_${PROJECT_NAME} MODULE ${SOURCES})
 
 target_include_directories(_${PROJECT_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/src")
-target_link_libraries(_${PROJECT_NAME} PRIVATE ngraph::ngraph)
+target_include_directories(_${PROJECT_NAME} PRIVATE "${FRONTEND_INCLUDE_PATH}")
```

Comment:
I think dependency from `ngraph::frontend_manager` should also add frontend includes.

Reply:
Even target_link_libraries(frontendmanager) is enough to add frontedn includes. So, removed this line